docs/pesin_satis.md

```diff
@@ -27,8 +27,8 @@ Aşağıdaki kod örneği ödeme isteğini *Client* nesnesi üzerindeki kurucu m
 $request = $client->createPaymentRequestBuilder()
     ->buildCard('4282209027132016', '123', 5, 2019)
     ->buildOrder('ORDERNO123456', '127.0.0.1')
-    ->buildBilling('John', 'Smith', 'test@test.net', '05321231212')
-    ->buildAndAddProduct('The Product', 'PR1', 1, 10)
+    ->buildBilling('John', 'Smith', 'test@test.net', '05321231212', null, null, null, null, null, 'TR')
```

Comment:
Country Code bilgisi ile ilgili bir sorun mu yaşadınız ? Geçtiğimiz günlerde bu konuda bir kişiden daha şikayet aldığımı hatırlıyorum. Eğer böyleyse bu bilgiyi PaymentRequestBuilder::buildBilling() metodundaki countryCode parametresine default value olarak verebilirsiniz. Ödeme sağlayıcıları kullanıcının email dışındaki bilgilerini opsiyonel olarak istiyorlar. Bu noktada bu kadar parametreyi sadece countryCode girdirmek için girdirmek çok mantıklı değil.